fe/src/main/java/org/apache/doris/http/rest/TableQueryPlanAction.java

```diff
@@ -146,15 +145,8 @@ protected void executeWithoutPassword(ActionAuthorizationInfo authInfo, BaseRequ
                                                  "only support OlapTable currently, but Table [" + tableName + "] "
                                                     + "is not a OlapTable");
                 }
-                // construct ConnectContext  for analyzer
-                ConnectContext context = new ConnectContext(null);
-                context.setCatalog(Catalog.getCurrentCatalog());
```

Comment:
Is it OK that catalog and cluster is not set?

Reply:
It OK, the ThreadLocal ConnectContext is already set before, with all necessary members.